### PR TITLE
example: add ability to drag clicked nodes from Migration example

### PR DESCRIPTION
**Pull request by larsmoa**
Allows dragging clicked nodes in the Migration example (by selecting the "Drag" option from the "Node inspection UI").

https://user-images.githubusercontent.com/6741854/170135799-6998ffcb-5cd5-4643-abe7-b44f9ecdf957.mp4

### CODE REVIEW

File: examples/src/utils/InspectNodeUi.ts [hunk 1]
```diff
@@ -55,11 +59,41 @@ export class InspectNodeUI {
...
+        
+        const gizmo = new TransformControls(this._viewer.getCamera(), this._viewer.renderer.domElement);
+        const offset = new THREE.Vector3();
+        const transform = new THREE.Matrix4();
+        gizmo.addEventListener('change', () => {
+          offset.copy(draggable.position);
+          offset.addScaledVector(startPosition, -1.0);
+          transform.makeTranslation(offset.x, offset.y, offset.z);
+          model.setNodeTransformByTreeIndex(node.treeIndex, transform);
+          console.log(node.treeIndex,  offset, transform.clone());
```
Comment (christjt) [inline examples/src/utils/InspectNodeUi.ts:78]: Remove this, creates insane amount of spam in the console.

**Review by christjt — APPROVED** [review]
This is very fun to play around with. Shockingly cool feature!

File: examples/src/utils/InspectNodeUi.ts (hunk 1 above)
Comment (christjt) [inline examples/src/utils/InspectNodeUi.ts:78]:
```suggestion
```

Comment (larsmoa) [reply examples/src/utils/InspectNodeUi.ts]: oops

**Pull request merged**